types/webpack/index.d.ts

```diff
@@ -919,7 +919,7 @@ declare namespace webpack {
     }
 
     class WatchIgnorePlugin extends Plugin {
-        constructor(paths: RegExp[]);
+        constructor(paths: string[] | RegExp[]);
```

Comment:
I don't think so, but I could imagine that in a lot of cases, it might be legitimately one or the other. That said, it's a common mistake to make. CC @RyanCavanaugh